index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.6.0/css/all.min.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <title>Capstone Home</title>
+</head>
+<body>
+  <header>
+    <nav class="mobileMenuToggle-container">
+      <i class="fa fa-bars" id="mobileMenuToggle"></i>
+    </nav>
+    <nav class="desktop-nav">
+      <div class="desktop-nav-first">
+        <ul class="desktop-nav-list">
+          <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+          <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+      <div class="desktop-nav-second">
+        <img src="images/riseup-short.png" alt="">
+        <ul class="desktop-nav-list2">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#speakers">Speakers</a></li>
+          <li><a href="#partners">Partners</a></li>
+          <li><a href="https://www.google.com/search?q=egypt+rise+up+summit&oq=egypt+rise+up+summit&aqs=chrome..69i57j69i59l3j0i22i30j69i64j69i60j69i61.16534j0j7&sourceid=chrome&ie=UTF-8">News</a></li>
+          <li><a href="#">R.UP Campaign</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <article class="home">
+    <div class="home-container">
+      <p class="home-welcome">"Hello! Sharing world"</p>
+      <h1 class="home-intro">Egypt RISEUP<br>GLOBAL SUMMIT<br>2022</h1>
+      <p class="home-desc">A joyful celebration believing in the value of openness and sharing, creating change with pepole form over 80 countries is taking place in october, in Egypt.</p>
+      <p class="home-time">2022.1.15(THU) ~ 16(FRI)</p>
+      <p class="home-lctn">@American University of Cairo, Conference Center </p>
+    </div>
+  </article>
```

Comment:
**Suggested Changes**
- [ ] Let us please increase the font size of the `h1` tag.
- [ ] After the above, let us please increase the font size of the `p` tags relative to the `h1` tag.